src/memory/AmdFlash.cc

```diff
@@ -251,7 +257,7 @@ uint8_t AmdFlash::peek(size_t address) const
 		} else {
 			return narrow_cast<uint8_t>(peekCFI(address));
 		}
-	} else if (state == State::PRGERR) {
+	} else if (state == State::STATUS || state == State::PRGERR) {
```

Comment:
This is perfectly fine. No need to change it. But if you prefer (personal preference) this could be an alternative:
`} else if (state == one_of(State::STATUS, State::PRGERR)) {`

Reply:
Will do.